browser/brave_wallet/BUILD.gn

```diff
@@ -8,9 +9,11 @@ source_set("brave_wallet") {
     "brave_wallet_delegate_impl.cc",
     "brave_wallet_delegate_impl.h",
   ]
+  sources += brave_browser_wallet_sources
```

Comment:
we'd better put all sources from `sources.gni` right there, there is no need to keep a separate file

Reply:
(cc @bridiver)